### ISSUE TITLE
docs: links to command help in appendix

### ISSUE THREAD

**Issue by hjoliver** [cylc/cylc-doc#8]
Depends on #2972 

See discussion in #3037

**Comment by oliver-sanders** [comment]
Duplicate of #124 

This is about doing this:

```rst
Hold tasks with :command:`cylc hold`.
```

Rather than this:

```rst
Hold tasks with ``cylc hold``.
```

The former creates a hyperlink to the relevant bit of the docs (blocked by #185) as well as providing a little integrity checking.
